packages/site-demo/content/product/design-tokens/colors/index.mdx

```diff
@@ -0,0 +1,52 @@
+import { ColorVariantsDesignTokenGroup }  from '@lumx/demo/components/design-tokens';
+
+# Colors
+
+**Generic colors to use within your application.**
+**Each color has different variants based on opacity.**
+
+## Primary palette
+
+### Blue
+
+Blue color is the primary color for your application.
+You can override it by defining another blue color palette in your scss file.
+You can also define another color as primary by overriding <code>\$lumx-color-primary-{variant}</code> variable.
```

Comment:
@matmkian I changed this sentence (and also for the secondary color), I'll let you review it